src/carvel/config/values/values.yml

```diff
@@ -112,5 +112,5 @@ scdf:
         enabled: false
         image:
           repository: micrometermetrics/prometheus-rsocket-proxy
-          tag: 2.0.0-M2
+          tag: 1.5.3
```

Comment:
Why is this in here? We are not downgrading PRP and if we were we would not do it in this PR to add an IT.

Reply:
I can revert these with a force push once all other changes are ready,